tasks/rh-sign-image/tests/test-rh-sign-image-single-component.yaml

```diff
@@ -97,6 +99,7 @@ spec:
               then
                 echo "pipeline_image does not match"
                 exit 1
-              fi             
+              fi
+              kubectl delete internalrequests --all
```

Comment:
see comment in other test

Reply:
nice! did not know we had this. I will update!

Reply:
fixed